src/Interceptor/Interceptor/CacheInterceptor.php

```diff
@@ -43,9 +69,13 @@ public function prefix(Instance $instance): Response
         $handler = $this->getHandler(CacheHandler::class, $annotation);
 
         if ($handler->contains($cacheKey) === false) {
+            self::$misses[] = $cacheKey;
```

Comment:
Could we move this logic into a dedicated wrapper that would have the responsibility to log misses and hits ? Like a `CacheInterceptorTracker` ?

Reply:
It will be useful only if we keep having this logic used in multiple classes, but since the legacy needs to disappear, I don't mind having duplicates for now